networkd/network_server.ml

```diff
@@ -1360,7 +1360,11 @@ module PVS_proxy = struct
   let configure_site _dbg config =
     debug "Configuring PVS proxy for site %s" config.site_uuid ;
     let call =
-      {Rpc.name= "configure_site"; params= [Rpcmarshal.marshal t.ty config]}
+      {
+        Rpc.name= "configure_site"
+      ; params= [Rpcmarshal.marshal t.ty config]
+      ; notif= false
```

Comment:
I think `notfif` as a name is unfortunate. Does this mean "notify" or "not if"? Where is this coming from?

Reply:
It means notify, your comments convinved marcello to change it to `notify` in rpclib 8, which is used on master.

I could backport the changes.